src/app.ts

```diff
@@ -46,10 +46,12 @@ async function app() {
       console.log('Additional information:');
       console.log('  [path-to-fsh-defs]');
       console.log('    Default: "."');
-      console.log('    If fsh/ subdirectory present, it is included in [path-to-fsh-defs]');
+      console.log('    If input/fsh/ subdirectory present, it is included in [path-to-fsh-defs]');
       console.log('  -o, --out <out>');
       console.log('    Default: "build"');
-      console.log('    If fsh/ subdirectory present, default output is one directory above fsh/');
+      console.log(
+        '    If input/fsh/ subdirectory present, default output is one directory above fsh/'
+      );
```

Comment:
Also here, this will change when `generated` is added.

Reply:
As I said above, I was planning to just change this when updating the output piece, but now I'm not sure if this is too weird to leave as is for now.